tests/phpunit/CRM/Event/Form/ParticipantTest.php

```diff
@@ -4,6 +4,7 @@
 use Civi\Api4\LocBlock;
 use Civi\Api4\Participant;
 use Civi\Api4\Phone;
+use Civi\Test\FormTrait;
```

Comment:
That class doesn't seem to exist... (hence the test failure)

Reply:
@totten ah right - I hadn't spotted that. It does fail locally though for other reasons - but I think resolving https://github.com/civicrm/civicrm-core/pull/27237 will make the solution there clearer